webpack.config.js

```diff
@@ -122,6 +122,7 @@ function lorisModule(mname, entries, override=false) {
 const config = [
   // Core components
   {
+    mode: 'production',
```

Comment:
I don't believe this line should be here.

Reply:
Is there a better place?

Reply:
The mode could be either production or development. I don't believe it should be statically written as production. Loris currently has an option in the project/config.xml whether in development or production mode. It would be best if we could grab the switch logic from there and then set depending on it.

Reply:
Didn't know that. Thanks, will look into it.